README.md

```diff
@@ -0,0 +1,65 @@
+# Google Cloud Storage
+
```

Comment:
Please put some warning message here; ie. `WARNING, initial draft! Not ready to use yet.` to make sure no one install it yet.